docs/config.rst

```diff
@@ -469,6 +469,24 @@ Here's an example of a target that will execute all "database" stacks::
       required_by:
         - databases
 
+
+Custom Log Formats
+------------------
+
+You may optionally provide a custom log format for stacker to use::
+
+  # optional log format overrides.
```

Comment:
Might be worth putting the defaults here, or at least explaining how we use the python logging module standard format variables (link to here? https://docs.python.org/2.7/library/logging.html#logrecord-attributes)